src/cantools/database/can/c_source.py

```diff
@@ -1361,66 +1384,77 @@ def _generate_choices_defines(database_name, messages, node_name):
     return '\n\n'.join(choices_defines)
 
 
-def _generate_frame_name_macros(database_name, messages, node_name):
+def _generate_frame_name_macros(database_name: str,
+                                messages: List["CodeGenMessage"],
+                                node_name: Optional[str]) -> str:
     result = '\n'.join([
         '#define {}_{}_NAME "{}"'.format(
             database_name.upper(),
             message.snake_name.upper(),
-            message.name)
+            message.message.name)
         for message in messages if _is_sender_or_receiver(message, node_name)
     ])
 
     return result
 
 
-def _generate_signal_name_macros(database_name, messages, node_name):
+def _generate_signal_name_macros(database_name: str,
+                                 messages: List["CodeGenMessage"],
+                                 node_name: Optional[str]) -> str:
     result = '\n'.join([
         '#define {}_{}_{}_NAME "{}"'.format(
             database_name.upper(),
             message.snake_name.upper(),
             signal.snake_name.upper(),
-            signal.name)
+            signal.signal.name)
```

Comment:
it would be nice if the `CodeGen{Signal,Message}` objects would also be renamed. maybe to something like `cg{signal,message}`? (ideally, the "forwarded" properties to regular `Signal`/`Message` objects should also be removed...)

Reply:
I added `cg_` everywhere.